src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -102,6 +102,9 @@
   {:outcome (schema/enum :success :not-found :error)
    :message schema/Str})
 
+(def OIDMappings
+  {schema/Str schema/Keyword})
+
```

Comment:
I agree that it's a bit weird. Saying that, it makes sense conceptually to me both ways (i.e., having the OID as the key in this case, or the shortname as the key in the case of the puppet short names map).

This is the format currently accepted by TK-Auth, so we'd have to change it there if we wanted this to be different. I'd be open to changing it so the OID is represented as a keyword, and the shortname is represented as either a string or a keyword. However, I think the mapping of oid -> shortname is the right way to do it, as that most closely matches the format of the `custom_trusted_oid_mapping` configuration file.

Maybe we should change the `puppet-short-names` map to match this format?